qemu/deps/block_device/repeat_blockdev_add.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+QEMU_IMG=qemu-img
+QSD=qemu-storage-daemon
+
+TMPD=/tmp/p291748
+mkdir -p $TMPD
+if ! which $QSD;then echo "$QSD does not exist";exit 1; fi
+
+rm -f ${TMPD}/qsd.pid
+
+"$QEMU_IMG" create -f qcow2 -F raw -b null-co:// ${TMPD}/top.qcow2
+
+(echo '{"execute": "qmp_capabilities"}'
+ sleep 1
+ while true; do
+     echo '{"execute": "blockdev-add", "arguments": {"driver": "qcow2", "node-name": "tmp", "backing": "node0", "file": {"driver": "file", "filename": "/tmp/p291748/top.qcow2"}}}'
+     echo '{"execute": "blockdev-del", "arguments": {"node-name": "tmp"}}'
+ done) | \
+"$QSD" \
+    --chardev stdio,id=stdio \
+    --monitor mon0,chardev=stdio \
+    --object iothread,id=iothread0 \
+    --blockdev null-co,node-name=node0,read-zeroes=true \
+    --nbd-server addr.type=unix,addr.path=${TMPD}/nbd.sock \
+    --export nbd,id=exp0,node-name=node0,iothread=iothread0,fixed-iothread=true,writable=true \
```

Comment:
I think that we should not make the fixed id for the `monitor`, `iothread`, `node-name` and `nbd`, since there may be a conflict with the id of the framework, such as the `iothread0`.
So shall we consider adding the arguments for those in this script? or naming those by random str to avoid the conflict.

Reply:
Using random temporary directory to avoid conflict.